### DISCUSSION TITLE
Did you mean feature

### DISCUSSION THREAD

**Discussion by kalrav12learning** [microsoft-search/pnp-modern-search#2970] [Q&A]
We have configured PnP Search in SharePoint Online. 
 
Please find the below problem statement - 

1)	Have configured spelling suggestion with {spelling Suggestion }} - but nothing is shown. 

Expected behavior – 

         Ability to include a “did you mean” – or similar function through PnP Search .

**Comment by wobba** [comment]
Look into the spelling options for the Microsoft Search provider https://microsoft-search.github.io/pnp-modern-search/usage/search-results/data-sources/microsoft-search/

**Comment by kalrav12learning** [comment]
Hi wobba 

Thanks for your reply. 

Yes, this spelling correction feature is available. 

As per this  "https://github.com/microsoft-search/pnp-modern-search/issues/697" blog, I have applied the "Did you mean" feature through the template by adding {{spellingSuggestion}} in the anchor tag but it is not recognizing and showing in search results. 

So can you please let me know what configuration steps I am missing here to add?
